src/Surfnet/StepupMiddlewareClientBundle/Identity/Dto/RaCandidateInstitution.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
```

Comment:
Please adjust the licence headers for the two newly created files

2019 and SURFnet B.V.

Reply:
updated